src/components/timers/Tabata.tsx

```diff
@@ -1,3 +1,145 @@
-const Tabata = () => null;
+import '../../styles/tabata.css';
+import '../../styles/settings.css';
+import { useState, useEffect, useRef } from 'react';
+import { ButtonOptions } from '../generic/Buttons';
+import { TabataStats } from '../generic/TabataStats';
+
+/*
+  component description
+    - renders a Tabata timer that consists of a work period, a rest period, and set number of rounds
+    - user inputs for the work, rest, and rounds are embedded in the component rather than wrapped - opportunity for enhancement in the future as time permits
+    - several state variables to keep track of time elapsed, round, and shift in phase between work and rest
+    - some sharing across other functions / components but not as much as I would have liked - also another opportunity as time permits
+*/
+
+const Tabata = () => {
+    const [rounds, setRounds] = useState<number>(0);
+    const [workTarget, setWorkTarget] = useState<number>(0);
+    const [restTarget, setRestTarget] = useState<number>(0);
+    const [phase, setPhase] = useState<string>('');
+    const [isActive, setIsActive] = useState<boolean>(false);
```

Comment:
if `isActive` = true, can `isPaused` ever be true? 

Can you find a way to refactor this to simplify to only one state variable?